cluster-provision/k8s/check-cluster-up.sh

```diff
@@ -18,7 +21,7 @@ function cleanup {
   cd "$DIR" && cd ../..
   export KUBEVIRTCI_PROVISION_CHECK=1
   export KUBEVIRT_PROVIDER="k8s-${provision_dir}"
-  export KUBEVIRT_NUM_NODES=2
+  export KUBEVIRT_NUM_NODES=3
```

Comment:
it affect all versions from now on, in this file, we want it also on < 19 ?

Reply:
I think we can leave it like that, this is only for testing.

Reply:
up to you, i just wonder how much more time it takes,
can you please update the description with the reason ?

Reply:
Agree, this is not relevant, let's remove it.